src/components/operations/operation-details/ko/runtime/operation-console.html

```diff
@@ -402,7 +402,7 @@ <h3>HTTP request
                     <i class="icon-emb icon-emb-eye"></i> Show
                     <!-- /ko -->
                 </button>
-                <button class="code-snippet-command" data-bind="copyToClipboard: $component.getPlainTextCodeSample"
+                <button class="code-snippet-command" data-bind="copyToClipboard: $component.getPlainTextCodeSample, click: function(data) {logCopyEvent();}"
```

Comment:
`"click: function(data) {logCopyEvent();}"` can be just `"click: logCopyEvent"`.

Reply:
Thank you, updated!